admin/en/source/pages/changelog.rst

```diff
@@ -10,7 +10,7 @@ Release Date: 2017-10-16
 
 New Features
 ~~~~~~~~~~~~
-* Using uforge-sync binary, users can now synchronize the target environment with a CentOS scan (without overlay).
+* Using uforge-sync binary, users can now synchronize the target environment with scans without overlay of CentOS 6, 7 and Debian 8 systems.
```

Comment:
What do you think of:
Using uforge-sync binary, users can now synchronize the target environment with a scan of CentOS 6, 7 or Debian 8 systems (without overlay).

Reply:
Or maybe `Using uforge-sync binary, users can now synchronize the target environment with scans (without overlay) of CentOS 6, 7 and Debian 8 systems.`. We need a third advice.

Reply:
I prefer Clement's option